EditorPluginExample/Resources/EditorData/Example.plugin.ts

```diff
@@ -0,0 +1,91 @@
+/// <reference path="../../typings/Atomic/AtomicWork.d.ts" />
+/// <reference path="../../typings/Atomic/EditorWork.d.ts" />
+
+const ExamplePluginUILabel = "Example Plugin";
+const ExamplePluginTBPath = "EditorData/Example.tb.txt";
+
+class ExamplePluginService implements Editor.HostExtensions.HostEditorService, Editor.HostExtensions.ProjectService {
+
+    name: string = "ExampleService";
+    description: string = "This service demonstrates plugin functionality functionality.";
+
+    private serviceLocator: Editor.HostExtensions.HostServiceLocator = null;
+    private extensionWindow: Editor.Modal.ExtensionWindow = null;
+
+    private helloLabel: Atomic.UITextField;
+    private nameField: Atomic.UIEditField;
+
+    initialize(serviceLoader: Editor.Extensions.ServiceLoader) {
```

Comment:
Another change I think may need to happen is renaming HostServiceLocator.uiServices to uiServiceRegistry, or renaming the UIService interface. It's slightly confusing (as evidenced by one of my teammates being a bit confused) as it currently stands.